gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dform_ttt2.lua

```diff
@@ -22,6 +22,9 @@ AccessorFunc(PANEL, "m_iSpacing", "Spacing")
 AccessorFunc(PANEL, "m_Padding", "Padding")
 
 local materialReset = Material("vgui/ttt/vskin/icon_reset")
+local materialLockOpen = Material("vgui/ttt/vskin/icon_lock_open")
+local materialLockClosed = Material("vgui/ttt/vskin/icon_lock_closed")
```

Comment:
I think lock icons can be tricky and interpreted im different ways i would not expect this to work for all toggle semantics. Can this rather be something that is more like checkmarks or something?

Reply:
I could do that, yes. However keep in mind that the icon can be set to anything, this is only the fallback (and I use it as a fallback here because it will be used that way for the role debugging)

Reply:
Yes but a sensible default would be good although a toggle button is rather pointless when a checkbox exists

Reply:
Done @saibotk